lib/sub_diff.rb

```diff
@@ -6,6 +6,7 @@
 require 'sub_diff/collection'
 require 'sub_diff/diff'
 require 'sub_diff/differ'
+require 'sub_diff/factory'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.